src/commands.py

```diff
@@ -75,6 +75,27 @@ def difficulties(ep):
     tabulate(sorted_diffi, headers="keys", tablefmt="orgtbl", stralign="center")))
 
 
+network_sort_keys = ["block", "member", "diffi", "uid"]
+
+
+def set_network_sort_keys(some_keys):
+    global network_sort_keys
```

Comment:
The problem is that the keys are used in the callback get_network_sort_key() for which I have no control (it's a callback...). So I somehow need a global variable. We could set it directly from silkaj.py but I figured that doing so through a method would be cleaner.